src/object/srv_obj.c

```diff
@@ -2872,6 +2872,19 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 			       &orw->orw_oid, NULL, 0, dtx_flags, NULL, &dth);
 		if (rc == 0) {
 			rc = obj_local_rw(rpc, &ioc, dth);
+			if (rc != 0)
+				DL_CDEBUG(rc == -DER_INPROGRESS || rc == -DER_TX_RESTART ||
+					      rc == -DER_REC2BIG ||
```

Comment:
Add the `rc == -DER_REC2BIG` test as it is logged at `DLOG_DBG` at line 1662.
From the NLT test result, `rc == -DER_REC2BIG` at line 2874 only when the statement `rc = -DER_REC2BIG` at line 1660 is previously done.